bff/test/Hosts.Tests/PageModels/PerComponentPageModel.cs

```diff
@@ -1,14 +1,10 @@
-﻿using Microsoft.Playwright;
+﻿using Microsoft.AspNetCore.Mvc.RazorPages;
```

Comment:
Is this meant to be part of the template changes?

Reply:
yes, as a side effect of adding the templates to aspire, the aspire tests became unstable. I had to make some changes to make the more stable. 

Theoretically I could split up the adding of the templates to aspire in a separate PR, but I wanted to test if it all worked.